extensions/desktop-common/desktop-common

```diff
@@ -0,0 +1,434 @@
+
```

Comment:
desktop-common needs to be marked executable (snapcraft will fail otherwise)

Reply:
it is now executable :-)